app/backend/src/couchers/servicers/auth.py

```diff
@@ -232,10 +232,13 @@ def CompleteSignup(self, request, context):
                 email=signup_token.email,
                 username=request.username,
                 name=request.name,
-                city=request.city,
                 gender=request.gender,
                 birthdate=birthdate,
                 hosting_status=hostingstatus2sql[request.hosting_status],
+                city=request.city,
+                geom=create_coordinate(request.lat, request.lng),
+                geom_radius=request.radius,
```

Comment:
No checks to see if a valid location was sent?

Reply:
These are not checked elsewhere either. I think postgis will complain if the given coordinate is invalid (or it might wrap it around, I'm not entirely sure).

For the radius, we might add database check constraints to enforce it being in some sensible range?

Reply:
I mean more, if for some reason it isn't included, the user will be at null island. Maybe that's fine and the frontend has responsibility for making sure it's set?

Reply:
I think that was the conclusion last time, I think someone was very vocal about not mixing serialization layer stuff with business logic... I'd be more than happy to add checks to make sure coordinates aren't on null island now though.

Reply:
Oh yes, that was me. I do still kinda think people should be *able* to have 0,0 as a location. But now I also think it's splitting hairs.

Reply:
Added some checks